lib/widgets/Picker.dart

```diff
@@ -1,15 +1,24 @@
 import 'package:flutter/material.dart';
 import 'package:numberpicker/numberpicker.dart';
 import 'package:random_run/theme.dart' as T;
+import 'package:bloc/bloc.dart';
+import 'package:random_run/bloc/picker.dart';
 
 class Picker extends StatefulWidget {
+  final int _wholeNumberValue;
+  final int _decimalNumberValue;
+  final Bloc _bloc;
+
+  Picker({@required String pickerValue, @required Bloc bloc})
+      : _wholeNumberValue = int.parse(pickerValue.split('.')[0]),
```

Comment:
does this still work for 26.2 miles?

Reply:
yea shouldn't be an issue since it splits only on `.`'s